### ISSUE TITLE
为什么eval里面的batch_size必须为1

### ISSUE THREAD

**Issue by cqray1990** [WenmuZhou/PytorchOCR#192]
'loader': {
            'type': 'DataLoader',
            'batch_size': 1,  # 必须为1
            'shuffle': False,
            'num_workers': 1,
            'collate_fn': {
                'type': 'DetCollectFN'

**Comment by novioleo** [comment]
crnn的可以长度不限，所以batchsize为1，不然你每个image的tensor size不同
